documentation/modules/auxiliary/scanner/http/dir_scanner.md

```diff
@@ -0,0 +1,54 @@
+## Description
+
+This module scans one or more web servers for interesting directories that can be further explored.
+
+## Verfication Steps
+
+1. Do: ```use auxiliary/scanner/http/dir_scanner```
+2. Do: ```set RHOSTS [IP]```
+3. Do: ```run```
+
+Let the default dictionary included in Metasploit set, set our target, and let the scanner run.
+
+## Scenarios
+
+**Running the scanner**
```

Comment:
instead of this line, if you are scanning a specific box (like metasploitable 2) call that out here so someone could reproduce the same results.  Typically we do this under triple # so that it makes a sub heading